WordPress/gutenberg
# Fix duotone rendering in site editor

## Description


Fix the site editor part of #35331

For duotone to work on the site editor we need to port the same changes introduced in the server at #34667.

## How has this been tested?




1. Set duotone styles in theme.json under `styles.blocks['core/image'].filter.duotone` as `var(--wp--preset--duotone--blue-red)` or a theme's duotone preset variable.
2. Open the site editor and add an image. 
3. The image should have duotone styles. 

## Screenshots 

Using TwentyTwentyTwo with the following added to theme.json:

```json
{
	"styles": {
		"blocks": {
			"core/image": {
				"filter": {
					"duotone": "var(--wp--preset--duotone--foreground-and-secondary)"
				}
			}
		}
	}
}
```

Before:

![site editor with image without the filter applied](https://user-images.githubusercontent.com/5129775/149230788-038d4939-eb09-4ca5-955f-f9b2e25ba302.png)

After:

![site editor with image with the filter applied](https://user-images.githubusercontent.com/5129775/149230800-003eb3d5-47f3-435f-9e15-0f5116ecdb4b.png)

## Types of changes





Bug fix

## Checklist:
